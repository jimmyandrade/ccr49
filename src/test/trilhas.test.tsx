import { create } from "react-test-renderer";
import TracksPage from "../pages/trilhas";

jest.mock("next/image", () => "Image");

describe("Tracks page", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<TracksPage />);
        expect(testRenderer).toMatchSnapshot();
    })
})