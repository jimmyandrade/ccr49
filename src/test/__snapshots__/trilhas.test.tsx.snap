// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tracks page should match snapshot 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <h2
    className="MuiTypography-root MuiTypography-h4"
  >
    Trilhas
  </h2>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiTypography-paragraph"
  >
    Ensino de 
    <i
      lang="en"
    >
      hard skills
    </i>
     por atividades lúdicas
  </p>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4"
    >
      <div
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiCardActionArea-root"
          href="/trilhas/preparacao-programa-aprendiz-estagiario"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
        >
          <div
            className="MuiCardMedia-root"
          >
            <Image
              alt="Quatro pessoas unindo as mãos, como um cumprimento de time"
              src="test-file-stub"
            />
          </div>
          <div
            className="MuiCardContent-root"
          >
            <h3
              className="MuiTypography-root MuiTypography-h6"
            >
              Preparação para o programa
              <br />
              de aprendiz e estagiário
            </h3>
          </div>
          <span
            className="MuiCardActionArea-focusHighlight"
          />
        </a>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4"
    >
      <div
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiCardActionArea-root"
          href="https://exame.com/invest/minhas-financas/7-musicas-que-ensinam-sobre-dinheiro/"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
        >
          <div
            className="MuiCardMedia-root"
          >
            <Image
              alt="Criança negra tocando violão. Do lado esquerdo, uma partitura."
              src="test-file-stub"
            />
          </div>
          <div
            className="MuiCardContent-root"
          >
            <h3
              className="MuiTypography-root MuiTypography-h6"
            >
              Aprendendo finanças
              <br />
              através da música
            </h3>
          </div>
          <span
            className="MuiCardActionArea-focusHighlight"
          />
        </a>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4"
    >
      <div
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <button
          className="MuiButtonBase-root MuiCardActionArea-root"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <div
            className="MuiCardMedia-root"
          >
            <Image
              alt="Criança negra segurando uma bola de futebol e fazendo sinal positivo com o dedo polegar."
              src="test-file-stub"
            />
          </div>
          <div
            className="MuiCardContent-root"
          >
            <h3
              className="MuiTypography-root MuiTypography-h6"
            >
              Paixão por futebol
              <br />
              e como encontrar um emprego
            </h3>
          </div>
          <span
            className="MuiCardActionArea-focusHighlight"
          />
        </button>
      </div>
    </div>
  </div>
</main>
`;
